json.go

```diff
@@ -93,35 +93,41 @@ func Protect(handler http.HandlerFunc) http.HandlerFunc {
 	}
 }
 
+// WrapJSONRequestHandlerFunc sets up an HTTP handler for use with JSON APIs with standard logging.
+// http.Requests will have a logger (with a request ID/method/path logged) attached to the Context.
+// This can be accessed via GetLogger(Context).
+func WrapJSONRequestHandlerFunc(w http.ResponseWriter, req *http.Request) {
```

Comment:
`WrapJSONRequestHandlerFunc` seems like the wrong name as it doesn't actually wrap a HandlerFunc.

This might work better as a `SetupRequestLogging(req *http.Request)` function since I think that bit is the most salvageable?